monitor/extensions/referencedata-wizards/src/main/resources/org/datacleaner/monitor/server/wizard/dictionary/simple/SimpleDictionaryReferenceDataPage.html

```diff
@@ -0,0 +1,29 @@
+<div style="min-height: 200px; background-repeat: no-repeat; background-position: bottom right; ">
+    <form action="repository/demo/jobs/multiple_upload" method="post" enctype="multipart/form-data">
```

Comment:
The path "repository/demo/jobs/multiple_upload" will only work in the default DC Monitor setup where we use the "demo" tenant. If you configure DC Monitor to run on another tenant (or set up a multi tenant config) things will break down.